app/controllers/api_controller.rb

```diff
@@ -223,7 +223,25 @@ def validate_include_params
   end
 
   def included_relationships
-    IncludeParamHandler.new(params).call
+    @included_relationships ||= IncludeParamHandler.new(params).call
+  end
+
+  def included_db_relationships
+    @included_db_relationships ||= included_relationships.map { |value| cast_relationship_to_symbol_hash(value) } if included_relationships.present?
+  end
+
+  def cast_relationship_to_symbol_hash(value)
```

Comment:
I guess the responsibility of this method is to convert relationships that are a list of query param includes to those compatible with `ActiveRecord::Base#includes` and `ActiveRecord::Base#joins` calls?  How do you feel about renaming this to `query_include_to_active_record_include_hash` or equivalent?

Reply:
For reference the gem we use does this, too: https://github.com/jsonapi-serializer/jsonapi-serializer/blob/master/lib/fast_jsonapi/serialization_core.rb#L155-L161. I'm confused why it doesn't explode 😕  re our need to special snowflake the flags thingy majig.